pkg/yurttunnel/agent/cmd.go

```diff
@@ -100,6 +101,10 @@ func (o *YurttunnelAgentOptions) validate() error {
 		return errors.New("--node-name is not set")
 	}
 
+	if o.nodeIP == "" {
+		return errors.New("--node-ip is not set")
```

Comment:
how about get nodeIP for NODE_IP env when `o.nodeIP` is empty, so don't need to update the command parameter in yaml file.